src/pyhub/llm/base.py

```
@@ -309,6 +340,7 @@ def _ask_impl(
         *,
         choices: Optional[list[str]] = None,
         choices_optional: bool = False,
+        schema: Optional[Type["BaseModel"]] = None,
```

Ruff (0.11.9)

Undefined name `BaseModel`

(F821)